core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -266,11 +280,12 @@ object ConsumerPerformance {
           val messageAndMetadata = iter.next
           messagesRead += 1
           bytesRead += messageAndMetadata.message.length
+          val currentTimeMillis = System.currentTimeMillis
```

Comment:
Do you think calling `System. currentTimeMillis` for every message will impact performance evaluation results?